pc/protocols/usb_boot.c

```diff
@@ -324,14 +325,28 @@ usbBootError_t usb_find_device_with_bcd(unsigned idx, char *input_addr,
                         fprintf(stderr, "Found Address: %s - VID/PID %04x:%04x\n",
                                 input_addr, desc.idVendor, desc.idProduct);
                     }
-
                     libusb_ref_device(dev);
                     libusb_free_device_list(devs, 1);
                     if (bcdusb)
                         *bcdusb = desc.bcdUSB;
                     *device = dev;
                     devs = 0;
 
+                    int speed = libusb_get_device_speed(dev);
+                    char *speed_str[] = {"Unknown", "Low/1.5Mbps", "Full/12Mbps", "High/480Mbps", "Super/5000Mbps"};
+                    usb_speed_enum = speed;
```

Comment:
Myabe better to switch over libusbs enum and set your accordingly

Reply:
I didn't quite get this.